Proposta de novo layout para grade de palestras

brenoferreira opened the pull request:
A visualização da grade de palestras hoje não está muito boa para o usuário, principalmente para ler as descrições das mesmas. O usuário hoje precisa clicar no título da palestra e abrir um balão que não oferece muito espaço para o texto e deixa uma experiência ruim.

Para resolver esse problema, alterei o layout com `<table>` e proponho um layout mais simples, que oferece muito mais espaço para a descrição da palestra, sem balões e com o conteúdo mais fácil de ler.

Abaixo uma screenshot:

![screen shot 2015-05-10 at 12 34 17](https://cloud.githubusercontent.com/assets/707019/7554944/e84a4ecc-f710-11e4-934b-8330b582710d.png)

Para palestras com uma descrição mais longa, há um botão de "ler mais" para o usuário ler o resto da descrição. Assim, a grade fica mais compacta e uma palestra não ocuparia muito espaço vertical, possibilitando o usuário ver com mais rapidez a grade toda (os horários, títulos da palestra e palestrante, pelo menos). Isso fica bem útil ao visualizar a grade em dispositivos móveis, com telas menores.

Fixes #13

brenoferreira committed:
update layout grade de palestras

brenoferreira committed:
update layout grade de palestras

brenoferreira committed:
imagem de SP tamanho menor

brenoferreira committed:
imagem de SP tamanho menor

egomesbrandao commented:
Acho legal mudarmos o leiaute... Mas fica muito extenso. Não temos alguma solução, facilmente implementável, que envolva um efeito Sanfona? https://jqueryui.com/accordion/
Poderíamos mostrar em uma linha Dia, Hora, Palestra e palestrante... e abrindo a descrição. O que acha?

egomesbrandao committed:
Corrigido link mailto

Corrigido link mailto:dnad@dotnetarchitects.net, faltava o a letra 't' no final

brenoferreira committed:
melhoria de layout da grade parte 2

brenoferreira committed:
melhoria de layout da grade parte 2

brenoferreira committed:
mini-bio lucindo & gleicon

brenoferreira committed:
mini-bio lucindo & gleicon

brenoferreira committed:
c4p encerrado

brenoferreira committed:
c4p encerrado

brenoferreira commented:
@egomesbrandao alterei o layout para deixar mais compacto

![screen shot 2015-05-17 at 13 15 50](https://cloud.githubusercontent.com/assets/707019/7671148/f20561a0-fc96-11e4-997c-595383dfedb7.png)

![screen shot 2015-05-17 at 13 16 00](https://cloud.githubusercontent.com/assets/707019/7671150/fa7f21ae-fc96-11e4-8c6f-ab8a26f0b88f.png)

egomesbrandao commented:
Eu daria mais uma enxugada... Nome do autor ao lado da palestra... cada palestra "colada" na outra... Deixar bem enxuto... 

Outra coisa, os horários do café... não tem horário do café da tarde no sábado... revisa isso pelo menos, por favor, mantendo os horários iguais aos do ano passado.

brenoferreira committed:
ajuste layout palestras

brenoferreira committed:
ajuste layout palestras

brenoferreira committed:
removido script desnecessario

brenoferreira committed:
ajustes

brenoferreira committed:
merge
